### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by dbgrandi**
We had failing travis builds because it was trying to use an old/unsupported OS version.

cc/ @adamkaplan

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,5 +5,5 @@ before_install:
     - pod install --project-directory=Example
 script:
 - set -o pipefail
-- xcodebuild -workspace Example/DBGHTMLEntitiesExample.xcworkspace -scheme DBGHTMLEntitiesExample -sdk iphonesimulator -configuration Debug test -destination platform='iOS Simulator',OS='10.0',name='iPhone 7' | xcpretty -c
+- xcodebuild -workspace Example/DBGHTMLEntitiesExample.xcworkspace -scheme DBGHTMLEntitiesExample -sdk iphonesimulator -configuration Debug test -destination platform='iOS Simulator',OS='11.4',name='iPhone X' | xcpretty -c
```
Comment (adamkaplan) [inline .travis.yml]: Modern travis doesn't require XCPretty.
https://blog.travis-ci.com/2018-08-15-deprecating-xctool

**Review by adamkaplan — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (adamkaplan) [reply .travis.yml]:
```
If your .travis.yml has a script key that looks like this:

script: set -o pipefail && xcodebuild -project Foo.xcodeproj -scheme Foo -destination 'platform=iOS Simulator,OS=11.3,name=iPhone X' build test | xcpretty
you can now use the default script by setting a few keys:

xcode_project: Foo.xcodeproj
xcode_scheme: Foo
xcode_destination: platform=iOS Simulator,OS=11.3,name=iPhone X
```
Default script does what you'd expect. be sure to remove the entire `script` block in the Yaml

Comment (adamkaplan) [reply .travis.yml]: Example https://github.com/yahoo/YMCache/blob/master/.travis.yml#L2-L12

Comment (dbgrandi) [reply .travis.yml]: Sounds good. Trying that now.

**Review by adamkaplan — APPROVED** [review]

**Comment by dbgrandi** [comment]
Example project was a bit chunky, so I refactored it a bit. Removed ORStackView, and consolidated to a single VC.

**Review by adamkaplan — APPROVED** [review]
🎉

**Pull request merged**